lib_test/test_h2.ml

```diff
@@ -95,4 +95,62 @@ module Headers_tests = struct
     ]
 end
 
-let () = Alcotest.run "ocaml-h2 unit tests" [ "headers", Headers_tests.suite ]
+module Body_length_tests = struct
```

Comment:
I didn't see an assertion that a call to ``Response.body_length ~request_method:`HEAD`` returns `` `Fixed 0``

Reply:
I don't know how I managed to do that